source/_integrations/ring.markdown

```diff
@@ -49,7 +51,9 @@ This integration does NOT allow for live viewing of your Ring camera within Home
 
 ## Binary sensor
 
-Once you have enabled the [Ring integration](/integrations/ring), you can start using a binary sensor. Currently, it supports doorbell, external chimes and stickup cameras.
+The binary sensor switches between off and on when motion, doorbell rings, and intercom unlock events occur.
+
+The binary sensor is being replaced with the event entity and you should migrate any automations to the event entity by release 2025.3.0.
```

Comment:
```suggestion
The binary sensor is being replaced with the event entity, and you should migrate any automations to the event entity by release 2025.3.0.
```

Reply:
Done and corrected the target date to 2025.4.0